internal/services/monitor/monitor_action_group_resource.go

```diff
@@ -58,6 +58,13 @@ func resourceMonitorActionGroup() *pluginsdk.Resource {
 
 			"resource_group_name": commonschema.ResourceGroupName(),
 
+			"location": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				Default:  "Global",
+				ForceNew: true,
+			},
```

Comment:
We have a commonschema function for `location` which we would typically use if this weren't a unique case, we should however still use parts of it. `Global` also needs to be lowercased since that's the normalisation location undergoes in the provider.

```suggestion
					"location": {
						Type:             schema.TypeString,
						Optional:         true,
						ForceNew:         true,
						Default:          "global",
						ValidateFunc:     validation.Any(
							location.EnhancedValidate,
							validation.StringInSlice([]string{
								"global",
							}, false),
						),
						StateFunc:        location.StateFunc,
						DiffSuppressFunc: location.DiffSuppressFunc,
					},
```

Reply:
Thanks for the reply @stephybun, I have added what you have wrote. I needed to add the schema dependency. 
	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"